tests/golden/defaults/openshift4-slos/openshift4-slos/appuio-ch-http-get-availability.yaml

```diff
@@ -3,6 +3,7 @@ kind: PrometheusRule
 metadata:
   annotations: {}
   labels:
+    monitoring.syn.tools/enabled: 'true'
```

Comment:
You should probably add

```
parameters:
  facts:
    distribution: openshift4
```
in `tests/defaults.yml` to have the default test case be the OpenShift case